### PR TITLE
feat(dragula-model): events after model synchronisation

### PR DESCRIPTION
**Pull request by DanielSchuech**
fix for [issue #12 ](https://github.com/bevacqua/angular-dragula/issues/12)

### CODE REVIEW

File: readme.markdown [hunk 1]
```diff
@@ -69,6 +69,8 @@ If your `ng-repeat` is compiled from array, you may wish to have it synced. For
 </ul>
 ```
 
+The standard `drop` event is fired before the model is synced. For that prupose you need to use the `drop-model`. The same behavior exists in the `remove` event. Therefore is the `remove-model` event. Further details are available under `Events`
```
Comment (kennethlynne) [inline readme.markdown]: prupose -> purpose

Comment (DanielSchuech) [inline readme.markdown]: Thanks man, corrected it

**Comment by bevacqua** [comment]
@luckylooke can you take a look at this?

**Comment by luckylooke** [comment]
Ok.. I will look at it soon.

**Comment by luckylooke** [comment]
Thank you @DanielSchuech! :+1:

**Pull request merged**